site/content/includes/quickstart-http-steps.md

```diff
@@ -0,0 +1,31 @@
+You can use the _{{% apiendpoint %}}json/write_ endpoint to write a tagged metric to M3 with the following data in the request body, all fields are required:
+
+-   `tags`: An object of at least one `name`/`value` pairs
+-   `timestamp`: The UNIX timestamp for the data
+-   `value`: The value for the data, can be of any type
```

Comment:
What do you mean by `can be of any type`?

Reply:
@benraskin92 string, int, float, etc… Is there a better way to put it?

Reply:
I would put float64 🤷

Reply:
@benraskin92 Is that always true? You may not use a string, or an int, or a boolean?